src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -49,7 +49,7 @@
     <Compile Include="**/*.cs" Exclude="Auth/*.cs;Correlation/*.cs;**/obj/**/*.cs" />
     <!-- Don't increase below versions without significantly testing on Functions V1!
          Increasing these versions increments some dependencies that have binding redirects in Functions V1. -->
-    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.Identity" Version="1.10.2" />
```

Comment:
the comment above about not increasing versions here makes me worry this might not be a safe update ^

Reply:
Since Functions V1 is not EOL yet, let's make sure we at least smoke test this with Functions V1 to confirm whether or not it's a breaking change.